hermes/src/store/benchmarks.rs

```diff
@@ -0,0 +1,109 @@
+use {
+    super::types::{
+        PriceFeedUpdate,
+        PriceFeedsWithUpdateData,
+        UnixTimestamp,
+    },
+    anyhow::Result,
+    base64::{
+        engine::general_purpose::STANDARD as base64_standard_engine,
+        Engine as _,
+    },
+    pyth_sdk::{
+        PriceFeed,
+        PriceIdentifier,
+    },
+    reqwest::Url,
+};
+
+const BENCHMARKS_REQUEST_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(30);
+
+#[derive(serde::Deserialize, Debug, Clone)]
+enum BlobEncoding {
+    #[serde(rename = "base64")]
+    Base64,
+    #[serde(rename = "hex")]
+    Hex,
+}
+
+#[derive(serde::Deserialize, Debug, Clone)]
+struct BinaryBlob {
+    pub encoding: BlobEncoding,
+    pub data:     String,
+}
+
+#[derive(serde::Deserialize, Debug, Clone)]
+struct BenchmarkPriceFeed {
+    pub binary: BinaryBlob,
+    pub parsed: PriceFeed,
+}
+
+impl TryFrom<BenchmarkPriceFeed> for PriceFeedUpdate {
+    type Error = anyhow::Error;
+
+    fn try_from(benchmark_price_feed: BenchmarkPriceFeed) -> Result<Self> {
+        let update_data = match benchmark_price_feed.binary.encoding {
+            BlobEncoding::Base64 => {
+                base64_standard_engine.decode(benchmark_price_feed.binary.data)?
+            }
+            BlobEncoding::Hex => hex::decode(benchmark_price_feed.binary.data)?,
+        };
```

Comment:
Readability nitpick.

```suggestion
        let data = &benchmark.binary.data;
        let update_data = match benchmark.binary.encoding {
            BlobEncoding::Base64 => base64_standard_engine.decode(data)?,
            BlobEncoding::Hex => hex::decode(data)?,
        };
```